release/release_gold.sh

```diff
@@ -20,23 +34,29 @@ function copy_rc {
 	ssh -i lrdcom-vm-1 root@lrdcom-vm-1 cp -a "/www/releases.liferay.com/dxp/release-candidates/${LIFERAY_RELEASE_VERSION}-${LIFERAY_RELEASE_RC_BUILD_TIMESTAMP}" "/www/releases.liferay.com/dxp/${LIFERAY_RELEASE_VERSION}"
 }
 
+function get_release_properties_value {
+    grep "${1}" ${file} | cut -d'=' -f2
+}
+
 function main {
 	check_usage
 
 	lc_time_run copy_rc
+	lc_time_run add_tag_to_release
 }
 
 function print_help {
-	echo "Usage: LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=<timestamp> LIFERAY_RELEASE_VERSION=<version> ${0}"
+    echo "Usage: LIFERAY_RELEASE_RC_BUILD_TIMESTAMP=<timestamp> LIFERAY_RELEASE_VERSION=<version> GIT_REPO_OWNER=<git_username> GIT_PAT=<personal_access_token> ${0}"
+    echo ""
+    echo "The script reads the following environment variables:"
+    echo ""
+    echo "    LIFERAY_RELEASE_RC_BUILD_TIMESTAMP: Timestamp of the build to publish"
+    echo "    LIFERAY_RELEASE_VERSION: DXP version of the release to publish"
+	echo "    GIT_REPO_OWNER: The repository owner's git username"
+    echo "    GIT_PAT: The user's Personal Access Token"
```

Comment:
$GIT_PATH should be $GITHUB_PAT to be more accurate.
The same for $GIT_REPO_OWNER.

Reply:
It should be arranged in alphabetical order.